insights/tests/test_util.py

```diff
@@ -114,3 +115,44 @@ def test_deep_nest(self):
         with pytest.raises(AssertionError):
             b["stuff"]["abba"][0]["foo"] = "cake"
             unordered_compare(a, b)
+
+
+class TestParseTable(unittest.TestCase):
```

Comment:
@SteveHNH would you mind to remove this class declaration and just have the `test_parse_table` function?  All new tests should use `pytest` instead of `unittest`.  I believe all of your tests will work as a function instead of a class method and you are already using the `pytest` assert.